lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+module Logidze
+  module VersionedAssociation
+    def load_target
+      target = super
+
+      return target if inversed
+
+      time = owner.logidze_requested_ts
+
+      if target.is_a? Array
+        target.map! do |object|
+          object unless object.class.has_logidze?
```

Comment:
Why do we check for logidze here? We've already checked that in `#association` method.